node/sc/config.go

```diff
@@ -46,7 +46,7 @@ var logger = log.NewModuleLogger(log.ServiceChain)
 // DefaultConfig contains default settings for use on the Klaytn main net.
 var DefaultConfig = SCConfig{
 	NetworkId: 1,
-	MaxPeer:   50,
+	MaxPeer:   1, // Only a single main-bridge and sub-bridge pair is allowed.
```

Comment:
If the two config values ​​should always match, it would be better to share one default value.